### DISCUSSION TITLE
Cortados dentro de privado BTN25 ¿Sí o no?

### DISCUSSION THREAD

**Discussion by manudona92** [manudona92/MapantES#22] [Q&A]
Hace unas semanas experimentamos si era posible quitar los cortados del verde oliva de zonas residencias (residental del OSM). El resultado fue que no se pudo hacer. 

Viendo el ejemplo de Murcia, he visto que el verde oliva del BTN25 (Canteras, aeropuertos, campos de golf, otros...) queda por encima del negro de los cortados. ¿Verdad?. El tema es ¿Cómo lo veis? 

Desde mi punto de vista, quizá dejará el negro en esas zonas porque te dá más información de los taluds en la cantera por ejemplo...pero se abre una discusión, porque en entornos privados cuanto más simplificado esté mejor. Un poco de dilema.

![image](https://user-images.githubusercontent.com/76484741/103661384-fcba5980-4f6e-11eb-8223-89eefc9a5dab.png)

**Comment by agustincaballero** [comment]
En realidad sí que se puede quitar los cortados de las zonas verde oliva, aquí un ejemplo del pueblo de Valverde, en El Hierro:

![pullautus_veg mapant](https://user-images.githubusercontent.com/18677366/103763611-604d9100-501a-11eb-8073-baf6447ed7a1.png)

Esto lo tengo ya implementado y cuando renderice todo de nuevo se aplicará.

La cuestión aquí es que eso lo estoy haciendo en TODOS los verde oliva; habría que decidir en qué casos sí y en qué casos no (creando otro color)

Personalmente, es que no soy nada "fan" del verde oliva, creo que tapamos muchos detalles sin que realmente estemos seguros de que esa área es "privada". Mi opinión es que deberíamos ir a "mostrar" el terreno tal y como es, no a "calificarlo".

Otro factor es que en el OSM, las áreas landuse=residential suelen ser muy bastas. Un ejemplo: en la cuadrícula 6183204, en El Hierro, donde señala la flecha hay un campo de fútbol:

![image](https://user-images.githubusercontent.com/18677366/103764782-617fbd80-501c-11eb-84e6-27af5abb6947.png)

Pero bueno, que tampoco quiero reabrir el debate, jeje

**Comment by agustincaballero** [comment]
Tengo listo para hacer la prueba que comentas, Manu. Dejando minas, campos de golf, y zonas militares con cortados por encima. Es decir, todos excepto las zonas "urbanizadas", donde es más susceptible que los cortados sean ruido.

Esta noche lanzaré para que se renderice de nuevo todo lo que llevamos procesado, para así hacernos idea de la situación actual.

**Reply by manudona92** [reply]
Gracias Agustín, 

Creo que es un buen debate aún así. Por aqui voy dejando los pros y contras del verde oliva residential de OSM

Pros (+)
- La sábana de color verde oliva hace ocultar a los cortados y vegetación que esten por debajo de ella. Por ejemplo cortados (muchas veces ruido) en zonas urbanas , además manchas de verde 3 (no sólo árboles de patios, también propios edificios y porches no renderizados en la BTN25 y que en KP salen como verde 3)
- Añade simplificación en zonas urbanas, necesario en mapas de ISOM (no ISSOM) a 1:15K / 1:10K

Cons (-)

- La mala generalización y precisión de la digitalización en muchos casos. Dependemos de cartografia colaborativa (OSM)
- Omite información en zonas que la necesitamos (pistas deportivas, posibles áreas runables/mapeables, etc.)
- Etc. 

Mi opinión se basa sobre todo en la simplificación de elementos urbanos o peri urbanos, ya que la ISOM en zonas metropolitanas tiene su límite, por eso el verde oliva ayuda a simplificar. Es decir, incluso organizando un rogaine urbano o MTBO en zonas peri urbana, la escala principal es 1:10K, y tener detalles de polideportivos, parques pequeños, etc, es muy difícil. Creo que haciendo una prueba con zonas metropolitanas más grandes nos podría ayudar más (por ejemplo 20 cuadriculas de A Coruña y área metropolitana, Murcia y huerta de Murcia, Madrid, Barcelona) Y ver con OSM residential vs sin OSM residential (solo con el de la BTN25 que viene por defecto, además de canteras, etc.)

Bueno reabro el debate, aunque con las pruebas que hagamos lo podemos ver mejor. Ya dices cuando hagas las pruebas de los cortados

Gracias!

**Reply by jarufe** [reply]
Al usar el verde oliva estamos catalogando (aunque nos equivoquemos) Si pensamos en orientación, al poner verde oliva estamos diciendo que es privado; yo dejaría solo curvas y edificios. Para el futuro, con tu método de renderización, habría varias opciones que se podrían dar desde el cliente. Una podría ser que alguien pidiera la cuadrícula pero sin los verdes oliva, sabiendo esa persona que le interesa ver y trabajar con lo que hay debajo. Otra podría ser que esa persona cambiara el OSM para decir que ese área en concreto no es privado (un parque en medio de una zona residencial) Al volver a renderizar ya desaparece lo de encima y emerge lo de abajo.
Por eso yo no trataría de usar verde oliva a tope con todo lo que nos da OSM y BTN, sino con un conjunto mínimo que en general sepamos que da buen resultado.

**Comment by agustincaballero** [comment]
Bueno, creo que podemos postesgar esta decisión un poco, y lo vamos viendo. Voy a hacer alguna otra prueba que tengo en mente y os cuento

**Comment by agustincaballero** [comment]
Vale, he descubierto una clave: en OSM, lo que va marcado como landuse=residential tiene a veces una etiqueta place=town/village. Eso es cuando abarca toda la ciudad, que es a lo que yo me refería de que tapa muchas cosas. Yo quitaría esas y me quedaría con sólo "parcelas". Por ejemplo, Valverde, en El Hierro, está sólo con las parcelas y creo que queda bien.
De todas formas, esto nos está quedando un poco espeso, jejeje. Podríamos hablarlo en una reunión y nos centraríamos más.
